
The usual pow, ... builtin for x86 - but not for many others.

  - Rene Rebe <rene@rocklinux.org>

--- ./scribus/scribusview.cpp.vanilla	2004-06-12 15:23:03.058045128 +0200
+++ ./scribus/scribusview.cpp	2004-06-12 15:23:18.707666024 +0200
@@ -25,6 +25,7 @@
 #endif
 
 #include "pageback.h"
+#include <math.h>
 #include <qcolor.h>
 #include <qfont.h>
 #include <qfontmetrics.h>
--- ./scribus/scribus.cpp.vanilla	2004-06-12 15:25:20.734115184 +0200
+++ ./scribus/scribus.cpp	2004-06-12 15:25:47.676019392 +0200
@@ -29,6 +29,8 @@
 #include <qvbox.h>
 #include <cstdio>
 #include <cstdlib>
+#include <math.h>
+
 #include <dlfcn.h>
 #include <unistd.h>
 #include <iostream>
